From 5ad2497d897370f04ec154df621bec02d34ba3b8 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 26 Jan 2018 16:00:04 -0700
Subject: [PATCH 04/12] msm8974-common: ril: Makefile maintenance

* Add top-level Android.mk
* Invert libril/Android.mk build guard (build if BOARD_PROVIDES_LIBRIL
  is true)

Change-Id: I4eb6c2316d268c2f8045dbb909322b731df47d8c
---
 ril/Android.mk        | 17 +++++++++++++++++
 ril/libril/Android.mk |  3 ++-
 2 files changed, 19 insertions(+), 1 deletion(-)
 create mode 100644 ril/Android.mk

diff --git a/ril/Android.mk b/ril/Android.mk
new file mode 100644
index 0000000..5dcfbb5
--- /dev/null
+++ b/ril/Android.mk
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2018 The LineageOS Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include $(call first-makefiles-under,$(call my-dir))
diff --git a/ril/libril/Android.mk b/ril/libril/Android.mk
index 3d8062c..e23daae 100644
--- a/ril/libril/Android.mk
+++ b/ril/libril/Android.mk
@@ -1,6 +1,7 @@
 # Copyright 2006 The Android Open Source Project
+# Copyright 2018 The LineageOS Project
 
-ifneq ($(BOARD_PROVIDES_LIBRIL),true)
+ifeq ($(BOARD_PROVIDES_LIBRIL),true)
 
 LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
-- 
2.7.4

